plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeTypeMapping.java

```diff
@@ -392,6 +396,66 @@ private void testTimestamp(ZoneId sessionZone)
                 .execute(getQueryRunner(), session, trinoCreateAndInsert("test_timestamp"));
     }
 
+    @Test
+    public void testForcedMappingToVarchar()
+    {
+        SqlExecutor snowflakeSqlExecutor = TestingSnowflakeServer::execute;
+        snowflakeSqlExecutor.execute("CREATE TABLE tpch.test_forced_varchar_mapping(a ARRAY, g GEOGRAPHY)");
```

Comment:
Avoid using static table name. It leads to flaky issues especially in SaaS connectors. 
Please use TestTable instead. Same for testUnsupportedDataType.